src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -529,18 +529,24 @@ def ndr_eff_calculation(
 
                     neighbor_effective_retention = (
                         effective_retention_raster.get(ds_col, ds_row))
-                    if neighbor_effective_retention >= retention_eff_lulc:
```

Comment:
I thought there might be an edge case here if `neighbor_effective_retention` and `retention_eff_lulc` are both 0, i.e. both pixels are part of a stream. But the result is still the same! 🎉